lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -17,11 +17,14 @@
 from charms.kubernetes_charm_libraries.v0.multus import (
     KubernetesMultusCharmLib,
     NetworkAttachmentDefinition,
-    NetworkAnnotation
+    NetworkAnnotation,
+    NadConfigChangedCharmEvents,
 )
 
 class YourCharm(CharmBase):
 
+    on = NadConfigChangedCharmEvents()
```

Comment:
why using "on"?

Reply:
I checked some sample custom charm events implementations and they generally use `on`. It has a meaning that when this event occurred. Do you have any other suggestion that suits better than it ?

Reply:
"on" is an attribute of `CharmBase`.
https://ops.readthedocs.io/en/latest/#ops.CharmBase.on

Reply:
So, this seems the proper way to create an event handler to observe the CharmEvents. Is there any other issue here ?

Reply:
Yes, we are overriding on property here and we need to do this to observe a custom charm event.
Otherwise, only the default CharmEvents will be observed. By overriding this we can provide our custom CharmEvents to be observed.

on: CharmEvents = CharmEvents()  # type: ignore
    """This property is used to create an event handler using :meth:`Framework.observe`,
    and can be one of the events listed at :class:`CharmEvents`.
https://ops.readthedocs.io/en/latest/_modules/ops/charm.html#CharmBase.on

Reply:
I think I got it. Thank you :)